Makefile

```diff
@@ -398,6 +398,18 @@ endef
 		make all install; \
 	fi
 
+
+HOOK_BIN_DIR ?= /usr/libexec/oci/hooks.d/
+HOOK_DIR ?= /usr/share/containers/oci/hooks.d
+
+generate-seccomp.install: 
+	install -m 755 bin/oci-trace-hook $(HOOK_BIN_DIR)
+	cp cmd/podman/oci-trace-hook/run.json $(HOOK_DIR)
+	cp cmd/podman/oci-trace-hook/stop.json $(HOOK_DIR)
+
+generate-seccomp: 
```

Comment:
Let's rename this to `oci-trace-hook` and add it as a dependency to the `binaries` target.